src/puppetlabs/kitchensink/json.clj

```diff
@@ -20,7 +20,7 @@
 
 (defn- clj-time-encoder
   [data jsonGenerator]
-  (.writeString jsonGenerator (coerce/to-string data)))
+  (.writeString ^com.fasterxml.jackson.core.JsonGenerator jsonGenerator ^String (coerce/to-string data)))
```

Comment:
This line would fail if anyone ever removed whatever require/import was implicitly loading this class, so we generally try to not use fully qualified classnames in the code body... FWIW